ci/travis_script_java.sh

```diff
@@ -24,6 +24,11 @@ JAVA_DIR=${TRAVIS_BUILD_DIR}/java
 pushd $JAVA_DIR
 
 export MAVEN_OPTS="$MAVEN_OPTS -Dorg.slf4j.simpleLogger.defaultLogLevel=warn"
-mvn -B install
+if [ $ARROW_TRAVIS_JAVA_BUILD_ONLY == "1" ]; then
+    # Save time and make build less verbose by skipping tests and style checks
+    mvn -DskipTests=true -Dcheckstyle.skip=true -B install
```

Comment:
@pitrou you could use `--quiet` that will output nothing unless there is an error.  Want me to add that?  Also, if you still want to run tests here, using #2465 will change the log level to get rid of the debug messages.